README.md

```diff
@@ -1,5 +1,9 @@
 # ![logo](https://raw.githubusercontent.com/azerothcore/azerothcore.github.io/master/images/logo-github.png) AzerothCore
 ## AutoBalanceModule
 - Latest build status with azerothcore: [![Build Status](https://travis-ci.org/azerothcore/mod-autobalance.svg?branch=master)](https://travis-ci.org/azerothcore/mod-autobalance)
-
-This module is intended to scale based on number of players, instance mobs and bosses' health, mana, and damage.
+* Script Name: AutoBalance
+* Original Authors: KalCorp and Vaughner
+* Original Script Name: AutoBalance
+* Description: This script is intended to scale based on number of players, instance mobs & world bosses' health, mana, and damage.
+* Ported to AzerothCore as a module
+* For installation instructions follow same as https://github.com/azerothcore/mod-duelreset
```

Comment:
can you revert these changes to the README? Yehonal removed them on purpose
We will have to update the readme later anyway